milatools/cli/commands.py

```diff
@@ -563,36 +606,89 @@ def code(
+        if not any("--account" in flag for flag in alloc):
+            warnings.warn(
+                T.orange(
+                    "Warning: When using the DRAC clusters, you usually need to "
+                    "specify the account to use when submitting a job. You can specify "
+                    "this in the job resources with `--alloc`, like so: "
+                    "`--alloc --account=<account_to_use>`, for example:\n"
+                    f"mila code {path} --cluster {cluster} --alloc "
+                    f"--account=your-account-here"
+                )
+            )
 
     if command is None:
         command = os.environ.get("MILATOOLS_CODE_COMMAND", "code")
 
-    try:
-        check_disk_quota(remote)
-    except MilatoolsUserError:
-        raise
-    except Exception as exc:
-        logger.warning(f"Unable to check the disk-quota on the cluster: {exc}")
-
-    cnode = _find_allocation(
-        remote, job_name="mila-code", job=job, node=node, alloc=alloc
-    )
-    if persist:
-        cnode = cnode.persist()
-    data, proc = cnode.ensure_allocation()
+    if remote.hostname != "graham":  # graham doesn't use lustre for $HOME
+        try:
+            check_disk_quota(remote)
+        except MilatoolsUserError:
+            raise
+        except Exception as exc:
+            logger.warning(f"Unable to check the disk-quota on the cluster: {exc}")
+
+    vscode_extensions_folder = Path.home() / ".vscode/extensions"
+    if vscode_extensions_folder.exists() and no_internet_on_compute_nodes(cluster):
+        # Sync the VsCode extensions from the local machine over to the target cluster.
+        print(
+            f"Copying VSCode extensions from local machine to {cluster} in the "
+            f"background..."
+        )
+        # Async:
+        copy_vscode_extensions_process = make_process(
+            copy_vscode_extensions_to_remote,
+            cluster,
+            vscode_extensions_folder,
+        )
+        copy_vscode_extensions_process.start()
 
-    node_name = data["node_name"]
+        # Sync:
+        # copy_vscode_extensions_to_remote(
+        #     cluster,
+        #     local_vscode_extensions_folder=vscode_extensions_folder,
+        #     remote=remote,
+        # )
+
+    if node is None:
+        cnode = _find_allocation(
+            remote,
+            job_name="mila-code",
+            job=job,
+            node=node,
+            alloc=alloc,
+            cluster=cluster,
+        )
+        if persist:
+            cnode = cnode.persist()
+
+        data, proc = cnode.ensure_allocation()
+
+        node_name = data["node_name"]
+    else:
+        node_name = node
+        proc = None
+        data = None
 
     if not path.startswith("/"):
         # Get $HOME because we have to give the full path to code
         home = remote.home()
-        path = "/".join([home, path])
+        if path == ".":
+            path = home
+        else:
+            path = "/".join([home, path])
```

Comment:
I think there was some issue with paths with a trailing `/.`  Windows, but I might be wrong. You're probably right that this is unnecessary. I simplified this if-statement just a bit in 685dc6c131d3bf3ee29032c14922006e6209c560 anyway

Reply:
You're probably right that this is unnecessary. I think it might have had to do with issues with trailing `/.` on Windows, but I'm not sure anymore. It's probably the same for vscode. I simplified this if-statement a bit in 685dc6c131d3bf3ee29032c14922006e6209c560)